cpp/src/groupby/sort/group_single_pass_reduction_util.cuh

```diff
@@ -39,59 +39,40 @@ struct reduce_functor {
 
   template <typename T>
   static constexpr bool is_supported(){
-    if (cudf::is_numeric<T>())
-      return true;
-    else if (cudf::is_timestamp<T>() and 
-              (k == aggregation::MIN or k == aggregation::MAX))
-      return true;
+    if (k == aggregation::SUM)
```

Comment:
Suggest capitalizing `k` like you would `typename T` template parameter. Or spell out `kind`.